frontend/integration-tests/views/crud.view.ts

```diff
@@ -116,15 +116,26 @@ export const actionsButton = actionsMenu.$('[data-test-id="actions-menu-button"]
 export const actionsDropdownMenu = actionsMenu.$('[data-test-id="action-items"]');
 
 export const resourceTitle = $('#resource-title');
-
 export const nameFilter = $('.form-control.co-text-filter');
 export const messageLbl = $('.cos-status-box');
 export const modalAnnotationsLink = $('[data-test-id=resource-summary] [data-test-id=edit-annotations]');
 
+export const clickResourceNameLink = async function(itemName: string) {
+  const item = element(by.linkText(itemName));
+  await browser.wait(until.presenceOf(item));
+  await item.click();
+};
 export const visitResource = async(resource: string, name: string) => {
   await browser.get(`${appHost}/k8s/ns/${testName}/${resource}/${name}`);
 };
 
+export const clickActionItem = async function(actionItem: string) {
```

Comment:
Use `clickDetailsPageAction` instead

Reply:
I thought of it, but the function needs to pass $(`[data-test-action="${label}"]`)  for `actionForLabel`, the element Action here does not have this attribute.